components/LibSwitcher.tsx

```diff
@@ -150,7 +150,7 @@ export default function LibSwitcher() {
         <SwitcherContent open={localIsOpen} callback={toggleSwitcher} />
       </SimpleModal>
       <div ref={ref} className={clsx('opacity-0 mx-1 capitalize', labelSizeClasses)}>
-        {query.slug[0].split('-').join(' ')}
+        {query.slug[0].replace(/\-/g, '')}
```

Comment:
I understand why you are doing this but I think it makes it way harder for someone who wants to help to understand what is happening

If I can in oss I always try to avoid regex